modules/redis/src/testFixtures/kotlin/com/loopers/testcontainers/RedisTestContainersConfig.kt

```diff
@@ -0,0 +1,22 @@
+package com.loopers.testcontainers
+
+import com.redis.testcontainers.RedisContainer
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class RedisTestContainersConfig {
+    companion object {
+        private val redisContainer = RedisContainer("redis:latest")
+            .apply {
+                start()
+            }
+    }
+
+    init {
+        System.setProperty("datasource.redis.database", "0")
+        System.setProperty("datasource.redis.master.host", redisContainer.host)
+        System.setProperty("datasource.redis.host.port", redisContainer.firstMappedPort.toString())
+        System.setProperty("datasource.redis.replicas[0].host", redisContainer.host)
+        System.setProperty("datasource.redis.replicas[0].port", redisContainer.firstMappedPort.toString())
+    }
+}
```

Comment:
Redis 테스트 컨테이너  Connection refused 오류납니다! 수정 필요해보입니다!

```suggestion
@Configuration
class RedisTestContainersConfig {
    companion object {
        private val redisContainer = RedisContainer("redis:latest")
            .apply {
                start()
            }

        init {
            System.setProperty("datasource.redis.database", "0")
            System.setProperty("datasource.redis.master.host", redisContainer.host)
            System.setProperty("datasource.redis.master.port", redisContainer.firstMappedPort.toString())
            System.setProperty("datasource.redis.replicas[0].host", redisContainer.host)
            System.setProperty("datasource.redis.replicas[0].port", redisContainer.firstMappedPort.toString())
        }
    }
}
```

Reply:
edf6bec 에서 수정하였습니다.